scripts/ci/run-ci-tests.sh

```diff
@@ -331,8 +331,11 @@ fi
 # more crit testing
 make -C test/others/crit run
 
-# coredump testing
-make -C test/others/criu-coredump run
+if [ "$UNAME_M" == "x86_64" ]; then
+	# coredump testing only works on x86_64
+	# the criu-coredump script is only x86_64 aware
+	make -C test/others/criu-coredump run
+fi
```

Comment:
I've opened an issue for this feature (https://github.com/checkpoint-restore/criu/issues/2433).

Would it be possible to move this check in `test/others/criu-coredump/test.sh`?